scripts/cfgdocgen/configuration.md

```diff
@@ -0,0 +1,46 @@
+---
+description: Telliot tweaks and settings to keep your rig running smoothly.
+---
+
+# Configuration reference
+
+## CLI reference
+
+Telliot commands and config file options are as the following:
+
+#### Required Flags <a id="docs-internal-guid-d1a57725-7fff-a753-9236-759dd3f42eed"></a>
+
+* `--config` \(path to your config file.\)
+
+#### Telliot Commands
+* `--logConfig` \(location of logging config file; default path is current directory\)
+{{range .CliDocs}}
+* `{{ .Name }}` {{range .Arguments }}\({{.Name}}{{if .Optional}}\(optional\){{end}}{{if .Help}}: {{ .Help }}{{end}}\){{end}}  \({{ .Help }}\)
```

Comment:
Can you explain how all this works and maybe add comments for someone else looking at this later.

Reply:
Sure I will add some comments.